tests/pytest/CMakeLists.txt

```diff
@@ -17,3 +17,8 @@ add_custom_target(pytest
                   COMMAND PYTHONPATH=${STAGED_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}${PYMOD_INSTALL_LIBDIR}
                           ${PYTHON_EXECUTABLE} -m pytest -rws -v --capture=sys ${CMAKE_CURRENT_SOURCE_DIR}
                   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+
+add_custom_target(pytest-tddft
+                  COMMAND PYTHONPATH=${STAGED_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}${PYMOD_INSTALL_LIBDIR}
+                          ${PYTHON_EXECUTABLE} -m pytest -rws -v -m tddft --capture=sys ${CMAKE_CURRENT_SOURCE_DIR}
+                  WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
I will remove this, I didn't mean to commit it. Just a handy hook to run some of my tests and not all the smoke tests.